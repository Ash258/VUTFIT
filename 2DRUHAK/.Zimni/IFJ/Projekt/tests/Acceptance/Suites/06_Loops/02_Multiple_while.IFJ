' Multiple while loop

scope
	dim a as integer = 0 - 25
	do while a <= 10
		print 2 * a + 0.02;
		a = a + 8
	loop
	a = (0 - 200 + 300)
	do while a >= 0
		print a;
		a = a - 11
	loop
end scope