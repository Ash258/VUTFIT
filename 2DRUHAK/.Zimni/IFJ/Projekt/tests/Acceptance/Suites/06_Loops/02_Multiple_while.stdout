-49.98-33.98-17.98-1.98 14.02 100 89 78 67 56 45 34 23 12 1